### ISSUE TITLE
Karma doesn't compile with CMSW_11_X

### ISSUE THREAD

**Issue by HerrHorizontal** [dsavoiu/Karma#4]
Trying to build Karma within CMSSW_11_X (gcc820) crashes printing the error:
```
In file included from /portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/ZJetAnalysis/interface/NtupleFlatOutput.h:12,
                 from /portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/ZJetAnalysis/src/NtupleFlatOutput.cc:1:
/portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/Common/interface/Analyzers/NtupleFlatOutputAnalyzerBase.h:23:10: fatal error: FWCore/Utilities/interface/TypeWithDict.h: No such file or directory
 #include "FWCore/Utilities/interface/TypeWithDict.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
>> Compiling edm plugin /portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/DijetAnalysis/src/NtupleHLTFilter.cc 
In file included from /portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/DijetAnalysis/interface/NtupleFlatOutput.h:12,
                 from /portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/DijetAnalysis/src/NtupleFlatOutput.cc:1:
/portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/Common/interface/Analyzers/NtupleFlatOutputAnalyzerBase.h:23:10: fatal error: FWCore/Utilities/interface/TypeWithDict.h: No such file or directory
 #include "FWCore/Utilities/interface/TypeWithDict.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
>> Compiling edm plugin /portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/DijetAnalysis/src/LeadingJetEtaFilter.cc 
In file included from /portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/ZJetAnalysis/interface/NtupleFlatOutput.h:12,
                 from /portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/ZJetAnalysis/src/NtupleFlatOutput.cc:1:
/portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/Common/interface/Analyzers/NtupleFlatOutputAnalyzerBase.h:23:10: fatal error: FWCore/Utilities/interface/TypeWithDict.h: No such file or directory
 #include "FWCore/Utilities/interface/TypeWithDict.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
gmake: *** [config/SCRAM/GMake/Makefile.rules:1742: tmp/slc7_amd64_gcc820/src/Karma/ZJetAnalysis/src/KarmaZJetAnalysis/NtupleFlatOutput.cc.o] Error 1
gmake: *** Waiting for unfinished jobs....
In file included from /portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/DijetAnalysis/interface/NtupleFlatOutput.h:12,
                 from /portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/DijetAnalysis/src/NtupleFlatOutput.cc:1:
/portal/ekpbms1/home/mhorzela/CMSSW_11_0_0_karma/src/Karma/Common/interface/Analyzers/NtupleFlatOutputAnalyzerBase.h:23:10: fatal error: FWCore/Utilities/interface/TypeWithDict.h: No such file or directory
 #include "FWCore/Utilities/interface/TypeWithDict.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

```

**Comment by dsavoiu** [comment]
It seems that the header file for `TypeWithDict` has been moved from
`FWCore/Utilities/interface/TypeWithDict.h`
to
`FWCore/Reflection/interface/TypeWithDict.h`
in CMSSW_11_X. Should be an easy fix, will look into this asap.

**Comment by dsavoiu** [comment]
Fixed on `master` (520980507161361b98db04aa8261e3658685aec9)
